conf/distro/include/ostro-xt-security-flags.inc

```diff
@@ -0,0 +1,3 @@
+SECURITY_CFLAGS_pn-crosswalk = "${SECURITY_NO_PIE_CFLAGS} ${SECURITY_PIC_CFLAGS}"
+SECURITY_LDFLAGS_remove_pn-crosswalk = "-fstack-protector-strong"
+SECURITY_LDFLAGS_append_pn-crosswalk = " -fstack-protector-all"
```

Comment:
**Solution 1**: Patch GYP so that it can distinguish between LDFLAGS for host and LDFLAGS for target, just like how CFLAGS are handled in GYP (via the `CFLAGS_host` environment variable). Since GYP has been deprecated by Chromium upstream, we have to implement it by ourselves.

**Solution 2**: I've reported [the issue](https://bugzilla.yoctoproject.org/show_bug.cgi?id=9948) to Yocto upstream, but it seems the fix cannot land in Yocto immediately, and it will take even longer to land in Ostro OS XT.

**Solution 3**: Tell users that GCC 4.9+ is required. I suppose the issue will gone when we migrate to GN in future releases.

**Solution 4**: Follow https://github.com/ostroproject/ostro-os-xt/blob/master/meta-ostro-xt/conf/distro/include/ostro-xt-security-flags.inc#L25 if the performance penalty is acceptable.

I'm tracking the issue in [XWALK-7114](https://crosswalk-project.org/jira/browse/XWALK-7114).